MDNetworking/Wrappers/MDDataConverters.cs

```diff
@@ -96,6 +96,46 @@ public bool AllowCachingOfConverter()
         }
     }
 
+    ///<summary> Data converter for types such as double or decimal</summary>
```

Comment:
In the future we should look at a ByteArray converter instead as it would be more optimal